tests/ImageSharp.Tests/Processing/Processors/Filters/LightnessTest.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using SixLabors.ImageSharp.PixelFormats;
+
+using Xunit;
+
+namespace SixLabors.ImageSharp.Tests.Processing.Processors.Effects
+{
+    using SixLabors.ImageSharp.Processing;
+    using SixLabors.ImageSharp.Tests.TestUtilities.ImageComparison;
+
+    [GroupOutput("Filters")]
+    public class LightnessTest
+    {
+        private readonly ImageComparer imageComparer = ImageComparer.Tolerant(0.007F);
+
+        public static readonly TheoryData<float> LightnessValues
+        = new TheoryData<float>
+        {
+            .5F,
+            .5F
```

Comment:
This is the same value twice. Why can it not be greater than 1?

Reply:
> 1 is the same as 1. Image will be white 100%